modules/perception/data/params/front_camera_extrinsics_low.yaml

```diff
@@ -0,0 +1,23 @@
+# Chery
```

Comment:
@xiaoshuliu remove the Chery note.

Reply:
@xiaoshuliu And why do you have both front_camera_extrinsics and  front_camera_extrinsics_low? The difference between these two are?

Reply:
@Capri2014 front_camera_extrinsics_low is for the case that camera is mounted inside the car (for most of the Chinese data). front_camera_extrinsics is for mkz8 in US.